zeebe/backup/src/main/java/io/camunda/zeebe/backup/metrics/BackupManagerMetrics.java

```diff
@@ -25,29 +31,27 @@ public class BackupManagerMetrics {
   private static final String LIST_OPERATION = "list";
   private static final String DELETE_OPERATION = "delete";
 
-  private static final Counter TOTAL_OPERATIONS =
-      Counter.build()
-          .namespace(NAMESPACE)
-          .name("backup_operations_total")
-          .help("Total number of backup operations")
-          .labelNames(LABEL_NAME_PARTITION, LABEL_NAME_OPERATION, LABEL_NAME_RESULT)
-          .register();
-  private static final Gauge OPERATIONS_IN_PROGRESS =
-      Gauge.build()
-          .namespace(NAMESPACE)
-          .name("backup_operations_in_progress")
-          .help("Number of backup operations that are in progress")
-          .labelNames(LABEL_NAME_PARTITION, LABEL_NAME_OPERATION)
-          .register();
-
-  private static final Histogram BACKUP_OPERATION_LATENCY =
-      Histogram.build()
-          .namespace(NAMESPACE)
-          .name("backup_operations_latency")
-          .help("Latency of backup operations")
-          .labelNames(LABEL_NAME_PARTITION, LABEL_NAME_OPERATION)
-          .buckets(0.01, 0.1, 1, 10, 60, 5 * 60)
-          .register();
+  private static final String BACKUP_OPERATIONS_TIMER_NAME =
+      NAMESPACE + "_backup_operations_latency";
+  private static final String BACKUP_OPERATIONS_IN_PROGRESS_NAME =
+      NAMESPACE + "_backup_operations_in_progress";
+  private static final MeterRegistry METER_REGISTRY = Metrics.globalRegistry;
```

Comment:
:x: : I would use the `MetricRegistry` that is associated to a partition, so it cannot be static. This will allow metrics to be unregistered when the registry is closed (for example a broker is not part of a partition anymore).